index.js

```diff
@@ -1,17 +1,14 @@
-/*eslint no-var:0, prefer-arrow-callback: 0, object-shorthand: 0 */
-'use strict';
-
-
-var Punycode = require('punycode/');
+import Punycode from 'punycode';
```

Comment:
Al estarlo importando así, en vez de estar usando el paquete instalado en el proyecto via npm, se sigue usando el punycode nativo de nodejs (deprecado desde hace muchas versiones), para usar el que estamos instalando, hay que importarlo así:

```suggestion
import punycode from 'punycode/punycode.js';

```
[Ver](https://github.com/mathiasbynens/punycode.js?tab=readme-ov-file#installation)

Además yo lo dejaría en lowercase en vez de capitalizado, sólo pq [lo exporta así](https://github.com/mathiasbynens/punycode.js/blob/main/punycode.js#L443). (habría que actualizarlo en las ocurrencias de más abajito, desde esta pantalla no me deja pq no contienen cambios)

Reply:
Well spotted!

WRT lowercasing var names, this we can do after this PR is merged. We've inherited a really oldschool style that came from hapi :see_no_evil: